### DISCUSSION TITLE
Is there any compatibility matrix with RabbitMQ ?

### DISCUSSION THREAD

**Discussion by PrzemyslawRodzik** [rabbitmq/rabbitmq-dotnet-client#1341] [Q&A]
Hi,
Currently in the company we use RabbitMQ version 3.8.9 and RabbitMQ.Client version 5.2.0. We would like to upgrade the library to version 6.4.0 and RabbitMQ to version 3.10.7. Could someone tell me if there is any information about the compatibility of particular versions of RabbitMQ.Client and RabbitMQ ? Unfortunately, I couldn't find anything on this topic.
I am mainly interested in whether the RabbitMQ.Client library in version 6.4.0 will be compatible or whether there may be problems with RabbitMQ in version 3.8.9 - quite old and 3.10.7.

**Comment by michaelklishin** [comment]
Any client version from the last few years can be used with any RabbitMQ version after 2.0. Obviously some features or behaviors will be RabbitMQ version-specific. But there were no wire protocol changes in years.

**Comment by michaelklishin** [comment]
Also worth mentioning: [RabbitMQ 3.8 has been entirely out of support](https://rabbitmq.com/versions.html) for nine months. Even 3.9 is out of general support, and 3.10 will follow suite in late July 2023.
